### ISSUE TITLE
wrong apt keyserver parameter

### ISSUE THREAD

**Issue by lombardialess** [MicrosoftDocs/microsoft-365-docs#5202]
In the "Add the Defender for Endpoint repository and key, add_apt_repo.yml:" section the apt key source is wrong:

WRONG:
- name: Add Microsoft APT key
  apt_key:
    keyserver: https://packages.microsoft.com/
    id: BC528686B50D79E339D3721CEB3E94ADBE1229CF
  when: ansible_os_family == "Debian"

CORRECT:
- name: Add Microsoft APT key
  apt_key:
    keyserver: https://packages.microsoft.com/keys/microsoft.asc
    id: BC528686B50D79E339D3721CEB3E94ADBE1229CF
  when: ansible_os_family == "Debian"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c5b7c531-90a1-7f6b-4c1d-0a126cdea398
* Version Independent ID: 5a10f1bb-6407-d706-0b79-bf448a8eda9d
* Content: [Deploy Microsoft Defender for Endpoint on Linux with Ansible](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/linux-install-with-ansible?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/linux-install-with-ansible.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/linux-install-with-ansible.md)
* Product: **m365-security**
* Technology: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

**Comment by msbemba** [comment]
@lombardialess Thank you for your feedback. We have made changes to the document via PR#https://github.com/MicrosoftDocs/microsoft-365-docs/pull/5220

Thanks
Sri

**Comment by konstruktoid** [comment]
```yaml
name: Add Microsoft APT key
apt_key:
keyserver: https://packages.microsoft.com/keys/microsoft.asc
id: BC528686B50D79E339D3721CEB3E94ADBE1229CF
when: ansible_os_family == "Debian"
```

This isn't actually correct, since `https://packages.microsoft.com/keys/microsoft.asc` isn't a keyserver. See https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_key_module.html and a the PR https://github.com/MicrosoftDocs/microsoft-365-docs/pull/4809/files

@msbemba @yogkumgit

**Comment by msbemba** [comment]
@konstruktoid Thank you for the feedback. We have merged #4809 and the document is now updated.

**Comment by yogkumgit** [comment]
@lombardialess Hope this update is helpful for you. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.
